### PR TITLE
Document `drop` and `clear` in the specification

### PR DESCRIPTION
**Pull request by gasche**

### CODE REVIEW

**Comment by o-** [comment]
thanks, lgtm

**Comment by o-** [comment]
closes #102

File: doc/ir-semantics.text [hunk 1]
```diff
@@ -177,6 +179,14 @@ extended with the new heap fragment into a new heap `H'`.
        = (I, T, (H,E)[x ← v], E, succ I l)
     when v := eval P H E e
 
+  step   (clear x) (P, I, T, H, E, l)
+       = (I, T, (H,E)[x ← ⊥], E, succ I l)
+    when v := eval P H E e
```
Comment (mhyee) [inline doc/ir-semantics.text:184]: This line should be removed.

Comment (gasche) [reply doc/ir-semantics.text:184]: Good catch, fixed in 2c0fb2e.

**Pull request merged**